benefits/core/migrations/0001_initial.py

```diff
@@ -67,6 +67,7 @@ class Migration(migrations.Migration):
                 ("unverified_title", models.TextField()),
                 ("unverified_headline", models.TextField()),
                 ("unverified_blurb", models.TextField()),
+                ("eligibility_confirmed_headline", models.TextField()),
```

Comment:
Can you run `./bin/makemigrations.sh`? I think it'll just update the autogenerated comment at the top of the file.